### PR TITLE
Add branch BITs

### PR DESCRIPTION
**Pull request by GaiaZano05**
Add BITs for every subsystem and one to run them together.

### CODE REVIEW

**Comment by katzuv** [comment]
https://github.com/Galaxia5987/robot-2023/pull/19#issuecomment-1411069008

File: src/main/java/frc/robot/commandgroups/bits/CheckSwerve.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Ports;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import frc.robot.subsystems.drivetrain.SwerveModule;
+
+public class CheckSwerve extends SequentialCommandGroup {
+    public CheckSwerve(SwerveDrive swerveDrive) {
+        addCommands(
+                new InstantCommand(swerveDrive::vroom).withTimeout(8),
```
Comment (katzuv) [inline src/main/java/frc/robot/commandgroups/bits/CheckSwerve.java:14]: `vroom` is not an indicative name. This should be fixed in another branch.

File: src/main/java/frc/robot/commandgroups/bits/ZeroPositionSwerve.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        Arrays.fill(zeroStates, new SwerveModuleState());
+    }
+
+    @Override
+    public void execute() {
+        swerve.setStates(zeroStates);
+    }
+
+    @Override
+    public boolean isFinished() {
```
Comment (katzuv) [inline src/main/java/frc/robot/commandgroups/bits/ZeroPositionSwerve.java]: Use the `withTimeout()` decorator instead.

File: src/main/java/frc/robot/subsystems/drivetrain/SwerveModule.java [hunk 3]
```diff
@@ -171,7 +171,7 @@ public void stop() {
      */
     public void vroom() {
         driveMotor.set(ControlMode.PercentOutput, 1);
-        angleMotor.set(ControlMode.PercentOutput, 0.5);
+        angleMotor.set(ControlMode.PercentOutput, 0.2);
```
Comment (katzuv) [inline src/main/java/frc/robot/subsystems/drivetrain/SwerveModule.java:174]: Why was this changed in this branch?

File: src/main/java/frc/robot/subsystems/intake/ConstantsIntake.java [hunk 4]
```diff
@@ -3,8 +3,11 @@
...
     public static final double GEAR_RATIO = 1 / 26.07;
     public static final double TICKS_PER_DEGREE = (42 * GEAR_RATIO) / 360;
-//TODO: check whether the calculation is correct.
     public static final double kP = 0;
     public static final double kI = 0;
     public static final double kD = 0;
+
+    public static final double ANGLE_UP = 0;
+    public static final double ANGLE_DOWN = 0;
+
```
Comment (katzuv) [inline src/main/java/frc/robot/subsystems/intake/ConstantsIntake.java:12]: This should be added in another branch.

**Review by katzuv — COMMENTED** [review]
Not all BITs should be stored in the command groups package. BITs that test a single subsystem should leave in a BITs package inside the subsystem's commands package.

File: src/main/java/frc/robot/subsystems/drivetrain/SwerveModule.java (hunk 3 above)
Comment (GaiaZano05) [reply src/main/java/frc/robot/subsystems/drivetrain/SwerveModule.java:174]: Because we don't use the function anymore, it was ment to break in the swerve, I converted the use of it for BITs.
